benchmark/build.gradle.kts

```diff
@@ -16,10 +16,22 @@
  * along with this program. If not, see http://www.gnu.org/licenses/.
  */
 plugins {
-    id(libs.plugins.wire.android.library.get().pluginId)
+    id(libs.plugins.android.library.get().pluginId)
+    id(libs.plugins.kotlin.android.get().pluginId)
 }
 
 android {
+    defaultConfig {
+        minSdk = 23
+        compileSdk = 34
```

Comment:
Please use Android.Sdk.compile and Android.Sdk.min in here so it always matches the app

Reply:
Well, actually it's a standalone module, it has nothing related to the app. Better to keep it's own config, than rely on the project and we may break it again.